.github/workflows/mongodb-oplog-workflow.yml

```diff
@@ -8,14 +8,11 @@ on:
     paths:
       - 'support/checkstyle/**'
       - 'support/ide-configs/**'
-      - 'support/revapi/**'
       - 'debezium-api/**'
       - 'debezium-assembly-descriptors/**'
       - 'debezium-core/**'
       - 'debezium-embedded/**'
       - 'debezium-connector-mongodb/**'
-      - 'debezium-microbenchmark/**'
-      - 'debezium-e2e-benchmark/**'
       - 'debezium-scripting/**'
```

Comment:
What about the `debezium-scripting` module?

Reply:
Right, this isn't needed either (it's not a dependency of this module).